website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -44,19 +44,24 @@ Please read through the entire procedure before beginning the upgrade. There are
 
 2. Review for breaking changes between v0.33 and v0.37: If you are already running Karpenter v0.37.x, you can skip this step. If you are running an earlier Karpenter version, you need to review the [Upgrade Guide]({{<ref "upgrade-guide#upgrading-to-0320" >}}) for each minor release.
 
-3. Set environment variables for your cluster:
+3. Set environment variables for your cluster to upgrade to the latest patch version of the current Karpenter version you're running on:
 
     ```bash
     export KARPENTER_NAMESPACE=kube-system
-    export KARPENTER_VERSION=1.0.0
+    export KARPENTER_IAM_ROLE_ARN="arn:${AWS_PARTITION}:iam::${AWS_ACCOUNT_ID}:role/${CLUSTER_NAME}-karpenter"
     export AWS_PARTITION="aws" # if you are not using standard partitions, you may need to configure to aws-cn / aws-us-gov
     export CLUSTER_NAME="${USER}-karpenter-demo"
     export AWS_REGION="us-west-2"
     export AWS_ACCOUNT_ID="$(aws sts get-caller-identity --query Account --output text)"
     ```
 
-4. Update your existing policy using the following:
-   Notable Changes to the IAM Policy include `SSM:GetParameter` to `SSM:GetParametersByPath` and additional tag-scoping for the `eks:eks-cluster-name` tag for instances and instance profiles.
+4. Set environment variables for first upgrading to latest patch version of your current minor version's Karpenter controller
+
+    ```bash
+    export KARPENTER_VERSION=<latest patch version of your current v1beta1 minor version>
+    ```
+
+5. Update your existing policy using the following:
```

Comment:
We shouldn't have changed the IAM policy on a patch version so I don't think that it's necessary to ask users to walk through the IAM upgrade multiple times. Did we change things in the IAM policy within a minor version?